server/mediaserver.go

```diff
@@ -820,8 +822,32 @@ func (s *LivepeerServer) HandlePush(w http.ResponseWriter, r *http.Request) {
 	remoteAddr := getRemoteAddr(r)
 	ctx = clog.AddVal(ctx, clog.ClientIP, remoteAddr)
 
-	clog.Infof(ctx, "Got push request at url=%s ua=%s addr=%s bytes=%d dur=%s resolution=%s", r.URL.String(), r.UserAgent(), remoteAddr, len(body),
-		r.Header.Get("Content-Duration"), r.Header.Get("Content-Resolution"))
+	sliceFromStr := r.Header.Get("Content-Slice-From")
```

Comment:
Is it maybe possible to add some unit test for this part? Maybe in `push_test,go`?

Reply:
Added